docs/apis/plugintypes/atto/index.md

```diff
@@ -5,7 +5,7 @@ tags: []
 
 <Since versions={["2.7"]} issueNumber="MDL-43841" />
 
-Atto is a JavaScript text editor built specifically for Moodle. It is the default text editor in Moodle from 2.7 onwards, and is implemented as a standard Moodle [text editor plugin](https://docs.moodle.org/dev/Editors). Most of the code is written in JavaScript using YUI modules.
+Atto is a JavaScript text editor built specifically for Moodle. It is the default text editor in Moodle from 2.7 onwards, and is implemented as a standard Moodle [text editor plugin](./../../apis/subsystems/editor). Most of the code is written in JavaScript using YUI modules.
```

Comment:
This should be a link to the .md file, not a relatie link to a folder. This is eitehr `editor.md` or `editor/index.md`

Reply:
You're right Andrew! Thanks for checking and raising it! It should be fixed now :-)